explorer/src/routes/network-components.tsx

```diff
@@ -1,22 +1,33 @@
 import * as React from 'react';
-import { Switch, Route } from 'react-router-dom';
+import { Switch, Route, RouteComponentProps } from 'react-router-dom';
+import { BIG_DIPPER } from 'src/api/constants';
 import { PageGateways } from 'src/pages/Gateways';
 import { PageMixnodeDetail } from 'src/pages/MixnodeDetail';
 import { PageMixnodes } from '../pages/Mixnodes';
 
 export const NetworkComponentsRoutes: React.FC = () => (
-  <Switch>
-    <Route exact path="/network-components/mixnodes">
-      <PageMixnodes />
-    </Route>
-    <Route path="/network-components/mixnodes/:id">
-      <PageMixnodeDetail />
-    </Route>
-    <Route path="/network-components/gateways">
-      <PageGateways />
-    </Route>
-    <Route path="/network-components/gateways/:id">
-      <h1> Specific Gateways ID</h1>
-    </Route>
-  </Switch>
+  <>
+    <Switch>
+      <Route exact path="/network-components/mixnodes">
+        <PageMixnodes />
+      </Route>
+      <Route path="/network-components/mixnodes/:id">
+        <PageMixnodeDetail />
+      </Route>
+      <Route path="/network-components/gateways">
+        <PageGateways />
+      </Route>
+      <Route
+        path="/network-components/validators"
+        component={(props: RouteComponentProps) => {
+          window.open(`${BIG_DIPPER}/validators`);
+          props.history.goBack();
+          return null;
+        }}
```

Comment:
Let's have a catch up with Mark about this part

Reply:
Can do. i chatted briefly with him yesterday about it.